File: melosys-eessi-app/src/test/java/no/nav/melosys/eessi/models/sed/medlemskap/impl/SvarAnmodningUnntakBeslutningTest.java
package no.nav.melosys.eessi.models.sed.medlemskap.impl;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class SvarAnmodningUnntakBeslutningTest {

    @Test
    void beslutningAvslag() {
        SvarAnmodningUnntakBeslutning beslutning = SvarAnmodningUnntakBeslutning.fraRinaKode("ikke_godkjent");

        assertThat(beslutning).isEqualTo(SvarAnmodningUnntakBeslutning.AVSLAG);
        assertThat(beslutning.getRinaKode()).isEqualTo("ikke_godkjent");
    }
}

File: melosys-eessi-app/src/test/java/no/nav/melosys/eessi/models/sed/medlemskap/impl/SvarAnmodningUnntakBeslutningTest.kt
package no.nav.melosys.eessi.models.sed.medlemskap.impl

import io.kotest.matchers.nulls.shouldNotBeNull
import io.kotest.matchers.shouldBe
import org.junit.jupiter.api.Test

class SvarAnmodningUnntakBeslutningTest {
    @Test
    fun beslutningAvslag() {
        val beslutning: SvarAnmodningUnntakBeslutning? = SvarAnmodningUnntakBeslutning.fraRinaKode("ikke_godkjent")

        beslutning.shouldNotBeNull().run {
            this shouldBe SvarAnmodningUnntakBeslutning.AVSLAG
            rinaKode shouldBe "ikke_godkjent"
        }
    }
}
